webpack.config.js

```diff
@@ -0,0 +1,29 @@
+/* eslint-disable */
```

Comment:
- [ ] @ezraermy its not good practise to disable linter check for the entire file, kindly try and fix the linter errors instead of disabling the check

Reply:
Dear @danmainah Thank you for your feedback again. 
![Screenshot from 2023-03-14 18-41-04](https://user-images.githubusercontent.com/35451409/225056452-0a32cd87-e297-4010-b31c-3a85b94c48a3.png)

Whenever I enabled eslint for 'webpack.congig.js' I encounter the error in the above pic. In my previous projects, I got some advice from another reviewer to disable it.